pkg/network/networknamescheme/networknamescheme.go

```diff
@@ -19,4 +19,35 @@
 
 package networknamescheme
 
+import (
+	"fmt"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
 const DefaultNetworkName = "eth0"
+
+func GetNetworkNameSchemeMap(vmi *v1.VirtualMachineInstance) map[string]string {
+	networkNameSchemeMap := map[string]string{}
+
+	mapDefaultNetworkToPodInterfaceName(vmi, networkNameSchemeMap)
+	mapMultusNonDefaultNetworksToPodInterfaceName(vmi, networkNameSchemeMap)
```

Comment:
I would prefer to see something where these helpers return the map rather than passing one via input arg then mutate it.